src/components/ForeCast.js

```diff
@@ -43,10 +43,10 @@ class ForeCast extends React.Component {
             <div className="forecast-3hour" key={index}>
               <p>{item.dt_txt.slice(10, -3)}</p>
               <div>
-                <img
+                <img className='forecast-icons'
                   src={`http://openweathermap.org/img/w/${
```

Comment:
Please replace the value of `src` here. We don't want to be using the other icons. Use the icons inside the `weather-icons` folder